android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AuthorizationsDao.java

```diff
@@ -139,4 +140,20 @@ public AuthRecord getAuthorization(@NonNull IdentityRecord identityRecord, @NonN
             return cursorToEntity(cursor, identityRecord);
         }
     }
+
+    @IntRange(from = 0)
+    @Override
+    public int purgeOldestEntries(@IntRange(from = 1) int identityId, @IntRange(from = 1) int maxOutstandingTokensPerIdentity) {
```

Comment:
Since `maxOutstandingTokensPerIdentity` is a constant value from `AuthIssuerConfig.java`, should we instead pass it to the constructor? Maybe this constructor should just take a reference to the `AuthIssuerConfig` instance itself, since it consumes two separate values from the class.

Reply:
Changed the constructor to take `AuthIssuerConfig`